docs/README.rst

```diff
@@ -0,0 +1,163 @@
+.. _readme:
+
+pycharm-formula
+============
```

Comment:
Headings markers should match heading text length, will be resolved when commit is added here:

https://github.com/myii/jetbrains-pycharm-formula/commit/a3bac46edc0678d4436b425e3929d5c04aea72f6#diff-8ab5636c6c18641e5c58358ef489e0f7R4